openmdao/devtools/xdsm_viewer/xdsm_writer.py

```diff
@@ -20,6 +20,15 @@
 
 from __future__ import print_function
 
+try:
+    from pyxdsm.XDSM import XDSM
+except ImportError:
+    msg = ('The pyxdsm package should be installed. You can download the package '
+           'from https://github.com/mdolab/pyXDSM')
+    raise ImportError(msg)
```

Comment:
any chance we could include the pip install command here?  Something like:

    msg = ('The pyxdsm package should be installed. You can download the package '
           'from https://github.com/mdolab/pyXDSM, or install using the pip command: '
           'pip install git+https://github.com/mdolab/pyXDSM.git')

Reply:
done